tests/migrations/test_writer.py

```diff
@@ -492,6 +499,33 @@ def upload_to(self):
         with self.assertRaisesMessage(ValueError, 'Could not find function upload_to in migrations.test_writer'):
             self.serialize_round_trip(TestModel2.thing)
 
+    def test_serialize_choices_iterator_should_unroll(self):
+        """An iterable marked to unroll should be converted to a list by
+        default."""
+        choices = [('a', 'A'), ('b', 'B')]
+
+        iter_choices = ImportableIterable()
+        iter_choices.unroll_in_migrations = True
+
+        class TestModel3:
+            thing = models.CharField(choices=iter_choices)
+
+        new_value = self.serialize_round_trip(TestModel3.thing)
+        self.assertEqual(new_value.choices, new_value._choices)
```

Comment:
Again, I think what you really want is `assertIsInstance` to check that `new_value.choices` has the proper data type. That applies to the other test function as well.